cp-overrides.yaml

```diff
@@ -5,6 +5,10 @@ modules:
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
                   ref: sprint-16
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules
+                  ref: sprint-16
```

Comment:
Did jboss.eap.logging.ext exist in sprint-16?

Reply:
No .. I think that was why I didn't add them previously, will correct again, sorry. (Juggling some builds at the same time, I should have just waited ;( )